CMake/ITKSphinxExamplesExternalData.cmake

```diff
@@ -20,17 +21,59 @@ list(APPEND ExternalData_OBJECT_STORES
   "${CMAKE_SOURCE_DIR}/.ExternalData"
   )
 
+set(ITK_SPHINX_EXAMPLES_DATA_RELEASED_ROOT_CID bafybeicjaaz3xi2qso3kdirmy5ixoxegvfp3bm7o44ytoh6egj5m6wgeuu)
```

Comment:
Is this per-release (i.e. 5.2, 5.3)? Could instructions please be added for when/how to update the content CID?

Reply:
This is per-release, e.g. 5.4. I will add instructions to the release process when the *ITK* repository is migrated.

Reply:
That works for me, thanks.